packages/gatsby-remark-prismjs/src/__tests__/highlight-code.js

```diff
@@ -1,5 +1,4 @@
 const parseLineNumberRange = require(`../parse-line-number-range`)
-const highlightCode = require(`../highlight-code`)
 
 describe(`highlight code and lines with PrismJS`, () => {
   it(`parses numeric ranges from the languages variable`, () => {
```

Comment:
@KyleAMathews Besides, I think these tests are some as, Why duplicate here?

https://github.com/gatsbyjs/gatsby/blob/ca12b4dedefd386943c725561037c0dcac44c4a6/packages/gatsby-remark-prismjs/src/__tests__/parse-line-number-range.js#L4-L43

Reply:
If there's duplicates, feel free to PR them away!